File: Python/hardware.py
from product import Product

# Kelas Hardware Turunan Kelas Product
class Hardware(Product):
    __brand = ""
    __model = ""

    # Constructor
    def __init__(self, brand = "", model = ""):
        self.__brand = brand
        self.__model = model

    # Setter
    def setBrand(self, brand):
        self.__brand = brand

    def setModel(self, model):
        self.__model = model

    # Getter
    def getBrand(self):
        return self.__brand
    
    def getModel(self):
        return self.__model

File: PHP/hardware.php
<?php

// Kelas Hardware Turunan Kelas Product 
class Hardware extends Product
{
    private $brand = "";
    private $model = "";

    // Constructor
    public function __construct($brand = "", $model = "")
    {
        $this->brand = $brand;
        $this->model = $model;
    }

    // Setter
    public function setBrand($brand)
    {
        $this->brand = $brand;
    }

    public function setModel($model)
    {
        $this->model = $model;
    }

    // Getter
    public function getBrand()
    {
        return $this->brand;
    }

    public function getModel()
    {
        return $this->model;
    }
}

?>